### PR TITLE
add rem-calc function and add path aliases

### PR DESCRIPTION
**Pull request by RostiMelk**

### CODE REVIEW

File: src/shared/pxToRem.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/**
+ * Convert px to rem
+ */
+export default function pxToRem(px: number | string, base = 16): string {
```
Comment (amfestoy) [inline src/shared/pxToRem.ts:4]: What is the difference between the library you have imported and this function? Do we need both?

File: src/stories/Foundation/Icon/index.tsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import * as icons from './components';
-import { getHexColor, Colors } from '../../../shared/colors';
+import { getHexColor, Colors } from '@shared/colors';
```
Comment (amfestoy) [inline src/stories/Foundation/Icon/index.tsx:3]: This turned out nice ⭐

File: src/stories/Foundation/Slider/index.tsx [hunk 3]
```diff
@@ -1,37 +1,43 @@
...
   background: transparent;
   -webkit-appearance: none;
-  transform: ${(p) =>
-    p.vertical ? `rotate(270deg) translate(-${p.length}px)` : ''};
-  transform-origin: ${(p) => (p.vertical ? 'top left' : '')};
+
   &::-webkit-slider-runnable-track {
-    height: ${(p) => (p.vertical ? '8px' : '8px')};
-    border-radius: 420px;
+    height: ${(p) => (p.vertical ? rem(8) : rem(8))};
```
Comment (amfestoy) [inline src/stories/Foundation/Slider/index.tsx:19]: I guess we can remove this check? if it is 8px anyway?

File: src/stories/Foundation/Slider/index.tsx [hunk 4]
```diff
@@ -1,37 +1,43 @@
...
   }
+
   &::-webkit-slider-thumb {
     box-sizing: border-box;
-    width: 22px;
-    height: 22px;
-    margin-top: ${(p) => (p.vertical ? '-7px' : '-7px')};
+    width: ${rem(22)};
+    height: ${rem(22)};
+    margin-top: ${(p) => (p.vertical ? rem(-7) : rem(-7))};
```
Comment (amfestoy) [inline src/stories/Foundation/Slider/index.tsx:29]: same. Btw this is my mistake earlier, but I appreciate you fixing it 🙏

File: src/stories/Foundation/Table/TableHeaderItem.tsx [hunk 5]
```diff
@@ -61,7 +62,7 @@ const TableHeaderItem = ({
         </Flex>
       </TH>
       {isSortable && align === 'right' && (
-        <TH align='right' width='30px'>
+        <TH align='right' width={rem(30)}>
```
Comment (amfestoy) [inline src/stories/Foundation/Table/TableHeaderItem.tsx:65]: Maybe double check this one since it is a prop?

**Review by amfestoy — APPROVED** [review]
(To be read in french accent) I approfe of thiz code, tres belle! 👨‍🎨

File: src/shared/pxToRem.ts (hunk 1 above)
Comment (RostiMelk) [reply src/shared/pxToRem.ts:4]: I'm sorry what? There is a lib?

File: src/stories/Foundation/Table/TableHeaderItem.tsx (hunk 5 above)
Comment (RostiMelk) [reply src/stories/Foundation/Table/TableHeaderItem.tsx:65]: Will do, but it shouuuuuld work :)))

**Comment by RostiMelk** [comment]
> (To be read in french accent) I approfe of thiz code, tres belle! 👨‍🎨

image
tyvm

**Pull request merged**